src/figuresClasses.ts

```diff
@@ -1,11 +1,89 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: Function;
```

Comment:
It's better to specify the type of the `getArea` method more explicitly. Instead of using `Function`, you should define it as `() => number` to indicate that it is a function returning a number.